lisa/src/main/java/it/unive/lisa/program/cfg/VariableTableEntry.java

```diff
@@ -276,4 +285,22 @@ public String toString() {
 	public CodeLocation getLocation() {
 		return location;
 	}
+
+	/**
+	 * Yields the annotations of this variable.
+	 * 
+	 * @return the annotations of this variable
+	 */
+	public Annotations getAnnotations() {
+		return annotations;
+	}
+
+	/**
+	 * Sets the annotations of this variable.
+	 * 
+	 * @param annotations the annotations to be set
+	 */
+	public void setAnnotations(Annotations annotations) {
```

Comment:
In some classes we have `addAnnotation`, in some other we have `setAnnotations`. We should unify them